list.js

```diff
@@ -0,0 +1,18 @@
+const arrayToList = (arr) => {
+    let list;
+    for (let i = 0; i < arr.length; i++) {
+        list = {
+            value: arr[i],
+            rest: list
+        }
+    }
+    return list
```

Comment:
The solution is fine, but It's a little bit hard to follow this list reassignment. This exercise is just perfectly suited for recursion.
```javascript
const toList = (value, ...values) => 
	({ value, rest: values.length === 0 ? void 0 : toList(values) })
```